src/bmffimage.cpp

```diff
@@ -452,6 +506,31 @@ uint64_t BmffImage::boxHandler(std::ostream& out /* = std::cout*/, Exiv2::PrintS
     case TAG_xml:
       parseXmp(buffer_size, io_->tell());
       break;
+    case TAG_brob: {
+      enforce(data.size() >= 4, Exiv2::ErrorCode::kerCorruptedMetadata);
+      uint32_t realType = data.read_uint32(0, endian_);
+      if (bTrace) {
+        out << "type: " << toAscii(realType);
+      }
+#ifdef EXV_HAVE_BROTLI
+      DataBuf arr;
+      brotliUncompress(data.c_data(4), data.size() - 4, arr);
+      if (realType == TAG_exif) {
+        uint32_t offset = arr.read_uint32(0, endian_) + 4;
```

Comment:
This `+4` could overflow, but it wouldn't cause anything particularly bad to happen. So this suggestion is optional:

```suggestion
        uint32_t offset = Safe::add(arr.read_uint32(0, endian_), 4u);
```